message.rst

```diff
@@ -13,6 +13,3 @@ This package contains various PSR-7 tools which might be useful in an HTTP workf
 * Message decorators
 * Message factory implementations for Guzzle PSR-7, Diactoros and Slim Framework.
 * Cookie implementation
-
-.. _message-factory:
```

Comment:
i am surprised sphinx did not complain about the duplicated reference target. it makes no sense to have this here, its in message-factory.rst